### PR TITLE
Negated assertions

### LINKED ISSUE #50: Support generating diffs with negated assertion
## Need
```gherkin
As a user
I want support for negated assertions
So that I can expect diffs and pass the test
```

## Solution
Support something like `expect(captureItem).to.not.be.identical` that will pass and also generate diffs.

### PR DESCRIPTION
**Pull request by adela-istrate-hs**
Closes #50

### CODE REVIEW

**Comment by adela-istrate-hs** [comment]
@andrei-picus-hs Can you take a look?

File: .gitignore [hunk 1]
```diff
@@ -1,3 +1,5 @@
+.DS_Store
```
Comment (andrei-picus-hs) [inline .gitignore]: Use a [global gitignore](https://help.github.com/articles/ignoring-files/#create-a-global-gitignore).

File: ui/components/details-diff.jsx [hunk 2]
```diff
@@ -79,7 +84,7 @@ class FailedTest extends Component {
 
   _renderSelectedView(paths) {
     const component =
-      _.find(FailedTest.VIEWS, {name: this.state.view}).component;
+      _.find(DetailsWithDiff.VIEWS, {name: this.state.view}).component;
```
Comment (andrei-picus-hs) [inline ui/components/details-diff.jsx]: Move the call on the previous line and split the arguments list.

**Comment by adela-istrate-hs** [comment]
Here is how it looks when a test passes but generates diffs:

screenshot at sep 15 16-12-38

**Comment by andrei-picus-hs** [comment]
We still have this failing setup scenario:

```
 1) Generate a dummy report for testing Second suite should fail with 'No browser provided' error:
     Error: No browser instance was provided
```

(in a later issue) Can we test that scenario only through unit/integration tests? It's preventing us from removing the `run setup || run acceptance` part.

**Comment by adela-istrate-hs** [comment]
@andrei-picus-hs can you review?

File: ui/styles/components/views.less [hunk 3]
```diff
@@ -1,10 +1,13 @@
 @import "../constants/colors";
 
-.simple img, .special{
+.simple img, .special {
   max-width: 100%;
   max-height: 100%;
   display: table;
   margin: 0 auto;
+  .clearfix {
```
Comment (andrei-picus-hs) [inline ui/styles/components/views.less]: Can't you target a different element here? The `.clearfix` trick is tied to `float` so we shouldn't depend on it for applying margins.

File: ui/styles/components/views.less [hunk 4]
```diff
@@ -5,8 +5,8 @@
   max-height: 100%;
   display: table;
   margin: 0 auto;
-  .clearfix {
-    padding-bottom: 10px;
+  input {
```
Comment (andrei-picus-hs) [inline ui/styles/components/views.less]: Blocks should be separated by a new line.

**Comment by adela-istrate-hs** [comment]
@andrei-picus-hs can I merge?

**Comment by andrei-picus-hs** [comment]
@adela-istrate-hs please see about this:

> Warning: `div` was passed a style object that has previously been mutated. Mutating `style` is deprecated. Consider cloning it beforehand. Check the `render` of `ImageDiff`. Previous style: {borderLeft: "1px solid #999", height: NaN, margin: 0, overflow: "hidden", position: "absolute", right: -2, width: NaN}. Mutated style: {borderLeft: "1px solid #999", height: NaN, margin: 0, overflow: "hidden", position: "absolute", right: -2, width: NaN}.

**Comment by adela-istrate-hs** [comment]
This comes from the `ImageDiff` package. I was thinking about making a contribution.

**Pull request merged**